src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/VirtualSecureModeEnclaveProviderBase.cs

```diff
@@ -237,7 +237,9 @@ private X509ChainStatusFlags VerifyHealthReportAgainstRootCertificate(X509Certif
                 chain.ChainPolicy.ExtraStore.Add(cert);
             }
 
+#pragma warning disable IA5352 // Certificates are not published to a Certificate Revocation List when revoked so this check cannot be made
             chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+#pragma warning restore IA5352
```

Comment:
This doesn't actually disable the warning from Roslyn so it can be removed if needed.